### PR TITLE
Fixes shortcodes ele2.9

### PR DESCRIPTION
**Pull request by crazypsycho**

### CODE REVIEW

File: trunk/Admin/css/dynamic-conditions-admin.css [hunk 1]
```diff
@@ -2,11 +2,3 @@
  * All of the CSS for your admin-specific functionality should be
  * included in this file.
  */
-
```
Comment (flammehawk) [inline trunk/Admin/css/dynamic-conditions-admin.css]: Should this file still be included? It is practicaly empty at this point in time

Comment (crazypsycho) [reply trunk/Admin/css/dynamic-conditions-admin.css]: Right, if we need it in future, we can readd it. So it is removed now

File: trunk/Lib/DynamicConditions.php [hunk 2]
```diff
@@ -148,7 +148,7 @@ private function defineAdminHooks() {
         $this->loader->addAction( 'elementor/element/popup/section_advanced/after_section_end', $pluginAdmin, 'addConditionFields', 10, 3 );
 
         $this->loader->addAction( 'admin_notices', $pluginAdmin, 'addAdminNotices', 10, 3 );
-        $this->loader->addAction( 'admin_enqueue_scripts', $pluginAdmin, 'enqueueStyles' );
+        #$this->loader->addAction( 'admin_enqueue_scripts', $pluginAdmin, 'enqueueStyles' );
         $this->loader->addAction( 'elementor/editor/before_enqueue_styles', $pluginAdmin, 'enqueueStyles' );
```
Comment (flammehawk) [inline trunk/Lib/DynamicConditions.php]: This line results in the plugin trying to load the same css file that was just removed. Comment it out or adjust.
trunk/Admin/DynamicConditionsAdmin.php and remove the css their. 
the last would be prefarable in my eyes.

Comment (crazypsycho) [reply trunk/Lib/DynamicConditions.php]: Sometimes i´m blind :-) Good that we do code-reviews.
It is commented out now.

**Pull request merged**